SharedPackages/BrowserServicesKit/Sources/Navigation/Extensions/WKFrameInfoExtension.swift

```diff
@@ -68,8 +68,10 @@ public extension WKFrameInfo {
             return "\(("\(file)" as NSString).lastPathComponent):\(line + 1)"
         }
 
+        let symbol = callingSymbol()
+
         // don‘t break twice
-        if Self.ignoredRequestUsageSymbols.insert(callingSymbol()).inserted {
+        if !symbol.contains("WebExtensionDynamicScripts") && Self.ignoredRequestUsageSymbols.insert(symbol).inserted {
```

Comment:
I was getting an assertion failure here - @mallexxx guided me to apply these changes to resolve.